lib/cli.rb

```diff
@@ -116,6 +116,14 @@ def self.handle_error(e)
     Cli.handle_error(e)
   end
 
+  def self.show_filter_note(scopes, filter)
+    scopes.each do |scope|
```

Comment:
Unused block argument - `scope`. You can omit the argument if you don't care about it.